docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,112 @@
+
+    This is a breaking change so it is highly recommended to upgrade your PMM to benefit from these enhancements. For more information see [Develop Advisors](https://docs.percona.com/percona-monitoring-and-management/details/develop-checks.html). 
+
+### Ubuntu 22.04 LTS support
+
+
+### Components upgrade
+  - **VictoriaMetrics**: VictoriaMetrics has been upgraded to [1.76.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.76.1).
+  - **Node exporter**: Node Exporter has now been updated to `1.1.X`.
+
+
+## New Features
+
+- [PMM-9749](https://jira.percona.com/browse/PMM-9849): Advisors: Possibility to run individual advisor checks separately
+
+- [PMM-9469](https://jira.percona.com/browse/PMM-9469): Advisors: Ability to have multiple queries in single check
+
+- [PMM-9468](https://jira.percona.com/browse/PMM-9468): Advisors: Ability to query VictoriaMetrics as a data source.
+
+
+## Improvements
+
+- [PMM-9839](https://jira.percona.com/browse/PMM-9839): Advisors: Fixed severity naming inconsistency for Critical checks.
+ 
+- [PMM-9841](https://jira.percona.com/browse/PMM-9841): Advisors: Advisor checks are now enabled by default.
+
+- [PMM-8326](https://jira.percona.com/browse/PMM-8326): Advisors: Changed the icon for the Edit Check Rule option with a more suggestive one, that better reflects this functionality.
+
+- [PMM-9362](https://jira.percona.com/browse/PMM-9362): Disconnect PMM and Portal: non-portal user can not perform disconnect
+
+- [PMM-9907](https://jira.percona.com/browse/PMM-9907): pmm2-client now supports Ubuntu 22.04 LTS.
+
+- [PMM-9780](https://jira.percona.com/browse/PMM-9780): VictoriaMetrics has been upgraded to 1.76.1. 
+
+- [PMM-5871](https://jira.percona.com/browse/PMM-5871): Node Exporter has now been updated to 1.1.x.
+
+- [PMM-9729](https://jira.percona.com/browse/PMM-9729): The Go version has now been updated to 1.18.
+
+
+
+## Bugs Fixed
+
+- [PMM-9854](https://jira.percona.com/browse/PMM-9854): Advisors: In some scenarios, PMM was not displaying the full list of advisors available for instances connected to Percona Platform. This issue is now fixed.
+
+
+- [PMM-8326](https://jira.percona.com/browse/PMM-8326): Advisors: Fixed text contrast issue on the **Failed Advisor Checks** page that was visible when navigating the list of results while using PMM with using the Light theme. 
+
+
+- [PMM-9426](https://jira.percona.com/browse/PMM-9426): DBaaS: Fixed an issue related to K8s monitoring where the k8s monitoring failed with k8s version 1.22 and higher.
+
+
+- [PMM-9363](https://jira.percona.com/browse/PMM-9363): Disconnect PMM and Portal: Logout user who disconnects pmm
+
+- [PMM-9885](https://jira.percona.com/browse/PMM-9885): Dashboard: Fixed the documentation links on the Advanced settings page on the PMM dashboard.
+
+- [PMM-9770](https://jira.percona.com/browse/PMM-9770): Fixed an issue where the query count displayed in PMM was incorrect and inconsistent.
```

Comment:
we also need [PMM-9828](https://jira.percona.com/browse/PMM-9828)
QAN Dashboard Navigator is not usable incase user redirected from a Dashboard ?